lib/egis/table_ddl_generator.rb

```diff
@@ -34,60 +34,28 @@ def column_definition_sql(columns)
       columns.map { |column| "`#{column.name}` #{column.type}" }.join(",\n")
     end
 
-    def serde?(format)
-      format.is_a?(Hash) && format.key?(:serde)
-    end
-
     def row_format_statement(format)
-      return serde_row_format_statement(format) if serde?(format)
-
-      delimited_row_format_statement(format)
-    end
-
-    def serde_row_format_statement(format)
-      <<-SQL
-        ROW FORMAT SERDE '#{format[:serde]}'
-        #{serde_properties(format)}
-        #{serde_input_format(format)}
-        #{serde_output_format(format)}
-      SQL
-    end
-
-    def serde_properties(format)
-      return '' unless format.key?(:serde_properties)
+      return "ROW FORMAT #{format}" if format.is_a?(String)
```

Comment:
As discussed the String option should also allow to define STORED AS option. I'd remove the "ROW FORMAT" from here and allow user to fill it in or not, for example if he just want to use "STORED AS PARQUET" option